Documentation/ColumnsConfig/Type/Category/Properties/ForeignTableWhere.rst

```diff
@@ -0,0 +1,17 @@
+.. include:: /Includes.rst.txt
+.. _columns-category-properties-foreign-table-where::
+
+=============
```

Comment:
The "=" above and below must be at least as long as the header text, but it can be longer. Otherwise there will be a rendering error.